tsl/src/nodes/gapfill/exec.h

```diff
@@ -96,6 +96,7 @@ typedef struct GapFillState
 	int64 gapfill_start;
 	int64 gapfill_end;
 	int64 gapfill_period;
+	Interval *gapfill_interval;
```

Comment:
would be good to add a comment about how gapfill_period and gapfill_interval are used.